src/functions/authoriser/application/__tests__/verifyEmployeeId.spec.ts

```diff
@@ -13,15 +14,48 @@ describe('verifyEmployeeId', () => {
       };
 
       try {
-        await verifyEmployeeId(verifiedToken);
+        await verifyEmployeeId(verifiedToken, employeeIdExtKey);
       } catch (err) {
         expect(err).toBe('Verified Token does not have employeeId');
       }
     });
+
+    it('should throw an exception when employeeid is an empty string', async () => {
+
+      employeeIdExtKey = 'employeeid';
+
+      const verifiedToken: VerifiedTokenPayload = {
+        sub: 'sub',
+        unique_name: 'unique_name',
+        employeeid: '',
+      };
+
+      try {
+        await verifyEmployeeId(verifiedToken, employeeIdExtKey);
+      } catch (err) {
+        expect(err).toBe('Verified Token does not have employeeId');
```

Comment:
This test would still pass in the event that an exception wasn't thrown. Return after `excpect` + explicit `fail()` after the `catch`?

Reply:
Oh dear, quite right, there are others like this, shall fix, stand by..